index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="  https://developers.google.com/fonts/docs/getting_started">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="logo">
+            <a href="#">Welcome</a>         
+        </nav>
+        <nav class="union-one">
+            <a>
+                <i class="fa-solid fa-bars"></i>
+            </a>
+        </nav>
+    </header>
+    <section id="headline-section">
+        <div class="headline-container">
+            <h1>Hey there. I'm Mariane <br>
+                <span>I'm a software developer </span></h1>
+                <p>
```

Comment:
[Optional] Please use an indentation of two spaces in all your documents. This is a convention and also [best practice.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)